### PR TITLE
Allow SelfSignedCertificates already provisioned

### PR DESCRIPTION
**Pull request by driosalido**
# Description

What - Allow SelfSignedCertificates already provisioned as secrets
Why - 
How -

## Type of change

Please leave one option from the following and delete the rest:

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: charts/port-ocean/values.yaml [hunk 1]
```diff
@@ -91,4 +91,7 @@ integration:
 
 selfSignedCertificate:
   enabled: false
-  certificate: ""
\ No newline at end of file
+  certificate: ""
+  secret: ""
+  useExistingSecret: false
+  key: crt
```
Comment (stavbernazport) [inline charts/port-ocean/values.yaml:97]: Just a minor issue over here

```suggestion
  secret: ""
    # name: ""
    # key: crt
```

**Review by stavbernazport — CHANGES_REQUESTED** [review]
Just a minor values issue, WDYT?

**Comment by driosalido** [comment]
> Just a minor values issue, WDYT?

I've just committed more changes that address the issue and another changed needed in the certificate secret

**Review by yairsimantov20 — APPROVED** [review]
🌊

**Review by stavbernazport — APPROVED** [review]

**Pull request merged**